content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -238,4 +238,25 @@ EOF
 )"
 
 ```
-{{% /capture %}}
\ No newline at end of file
+
+## `coredns` pods have `CrashLoopBackOff` or `Error` state
+
+If you have nodes that are running SELinux with an older version of Docker you might experience a scenario
+where the `coredns` pods are not starting. To solve that you can try one of the following options:
+
+- Upgrade to a [newer version of Docker](/docs/setup/independent/install-kubeadm/#installing-docker).
+- Disable SELinux.
```

Comment:
Shall we add some commands or links here to guide users how to disable this?

* Edit the `/etc/selinux/config` file to set the  `SELINUX` parameter to  `disabled`, and then reboot the server.
* Run the `setenforce 0` command.

Or add such a link on [disabling selinux](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security-enhanced_linux/sect-security-enhanced_linux-enabling_and_disabling_selinux-disabling_selinux).

Reply:
sounds good. :+1: 
i will add the link later today.